docs/guides/ai-transpiler-passes.mdx

```diff
@@ -23,6 +23,7 @@ The following passes are currently available:
  - `AICliffordSynthesis`: Clifford circuit synthesis
  - `AILinearFunctionSynthesis`: Linear function circuit synthesis
  - `AIPermutationSynthesis`: Permutation circuit synthesis
+ - `AIPauliNetworkSynthesis`: Pauli Network circuit synthesis
```

Comment:
Is it essential to capitalize Network? I'm ok with it if we stay consistent, though our general guidelines are to keep things lowercase unless branded or a product name (or are officially capitalized elsewhere in the world) 🌎

Reply:
In this case it should be capitalized to be coherent with the class name available in the code